_includes/pq-compression/overview-text.mdx

```diff
@@ -1 +1 @@
-[Product quantization (PQ)](/developers/weaviate/concepts/vector-index#hnsw-with-product-quantizationpq) is a form of data compression that reduces the memory footprint of a vector index. HNSW is an in-memory vector index, so enabling PQ for HNSW lets you work with larger datasets. For a discussion of how PQ saves memory, see [this concepts section](/developers/weaviate/concepts/vector-index#hnsw-with-compression).
+[Product quantization (PQ)](/developers/weaviate/concepts/vector-index#hnsw-with-product-quantizationpq) is a form of data compression that works with HNSW indexes. PQ reduces the memory footprint of a vector index, so enabling PQ for HNSW lets you work with larger datasets. For a discussion of how PQ saves memory, see [HNSW with compression](/developers/weaviate/concepts/vector-index#hnsw-with-compression).
```

Comment:
I believe PQ is not unique to HNSW. I think they're looking at allowing PQ and BQ available for either index. 

How about 
```
is a form of data compression for vectors. PQ reduces the memory ...
```

Reply:
Updated